src/views/messages-view.jsx

```diff
@@ -784,6 +805,164 @@ class MessagesView extends React.Component {
     uploader.cancel();
   }
 
+  handlePickReply(m) {
+    this.setState({reply: null})
+    //instance.setState({typingIndicator: false});
+    if (m) {
+      let cont = m.content
+      if (cont) {
+        if (typeof cont == 'string') {
+          cont = Drafty.init(cont);
+        }
+        cont = Drafty.preview(cont, 30, (src, target) => {
+          if (src.data && src.tp && src.tp == 'IM') {
+            // For images, copy val and ref fields.
+            ['val', 'ref'].forEach(key => {
+              const val = src.data[key];
+              if (val) {
+                target.data[key] = val;
+              }
+            });
+          }
+        });
+
+        let header = '';
+        const seq = m.seq;
+        const idx = this.msgIndex(seq);
```

Comment:
If message is not found, `msgIndex` returns 0. That probably is not the desired behavior.